server/sonar-server/src/main/java/org/sonar/server/ce/ws/TaskAction.java

```diff
@@ -120,7 +122,12 @@ public void handle(Request wsRequest, Response wsResponse) throws Exception {
 
   private void checkPermission(Optional<ComponentDto> component) {
     if (component.isPresent()) {
-      userSession.checkComponentPermission(SCAN_EXECUTION, component.get());
+      if (!userSession.hasOrganizationPermission(component.get().getOrganizationUuid(), GlobalPermissions.SYSTEM_ADMIN) &&
```

Comment:
put `component.get().getOrganizationUuid()` into a variable for readability, perf and to ensure the same default organization is checked (even though the implementation is cached, it's not in the contract of the interface)